server/api/user/user.controller.ts

```diff
@@ -189,6 +190,37 @@ class UserController {
     res.json(removedUser);
   }
 
+  async removeUser(req: ApiRequest<OrgEdipiParams>, res: Response) {
```

Comment:
Maybe these should be a bit more descriptive, such as `removeUserFromGroup` and `addUserToGroup`.